### ISSUE TITLE
NTR: Pancreatic isoamylase low in blood (> 3 years old) (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8206]
New term request Pancreatic isoamylase low in blood (> 3 years old) (segal_080822211715):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000721
- label: Decreased circulating pancreatic isoamylase activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:30675141
- def: The activity of alpha-Amylase (1,4-alpha-D-glucan-4-glucanohydrolase E. C. 3.2.1.1, pancreatic isoamylase) in the blood circulation is below the lower limit of normal.
- comment: Alpha-Amylase (1,4-alpha-D-glucan-4-glucanohydrolase E. C. 3.2.1.1.) catalyses the hydrolysis of starch, glycogen and related poly- and oligosaccharides. In acute pancreatitis the hyperamylasemia is characterised by an increased activity in plasma of pancreatic isoamylase.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8206
- synonyms:
